src/Npgsql/TypeMapping/INpgsqlTypeMapper.cs

```diff
@@ -40,7 +40,7 @@ public interface INpgsqlTypeMapper
     /// A component which will be used to translate CLR names (e.g. SomeClass) into database names (e.g. some_class).
     /// Defaults to <see cref="NpgsqlSnakeCaseNameTranslator"/>
     /// </param>
-    /// <typeparam name="TEnum">A CLR enum type to be mapped</typeparam>
+    /// <typeparam name="TEnum">The .NET enum type to be mapped</typeparam>
```

Comment:
FYI we use the term "CLR" a lot in the docs; though I don't really mind switching to talking about ".NET" (we're already not 100% consistent).

Reply:
@roji just to clarify - should I bring back the old "CLR" stuff for consistency or is it fine this way?